helm-chart/renku-core/templates/deployment.yaml

```diff
@@ -51,48 +56,48 @@ spec:
           {{ end }}
           env:
             - name: REDIS_HOST
-              value: {{ include "call-nested" (list . "redis" "redis.fullname") }}-master
+              value: {{ include "call-nested" (list $ "redis" "redis.fullname") }}-master
             - name: REDIS_PORT
               value: "6379"
             - name: REDIS_DATABASE
               value: "0"
```

Comment:
Are all versions of the service using the same redis db (ie for example by prefixing "their" keys)?

Reply:
Right. They don't prefix anything because a valid project only exists in one of those deployments and so there is only one redis key for it at any moment.

Reply:
The library that we use to interface with redis supports namespaces. We decided to add namespaces here to avoid surprises in future. There is still one redis instance but its keys are namespaced.